src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -200,9 +241,13 @@ def get_url(image):
                 )
             else:
                 logging.error("no image found in compose %s", compose_url)
+    elif centoshtml_url:
+        return centoshtml2image(centoshtml_url, "x86_64")
     else:
         logging.error(
-            "neither source nor compose specified" "in image %s", image["name"]
+            "neither source nor compose nor centoshtml specified"
+            "in image %s",
```

Comment:
With this improvement, we can just have
```
"centoshtml": "https://cloud.centos.org/",
```
in the config file; then it automagically download the latest centos stream image? Very nice!

Reply:
well `"centoshtml": "https://cloud.centos.org/centos/9-stream/x86_64/images",` but yes